app/controllers/application_controller.rb

```diff
@@ -21,6 +21,14 @@ def user_with_groups
   end
   helper_method :user_with_groups
 
+  def normalize_key(key)
```

Comment:
Can this be in the works controller rather than the application_controller since it's only used by the one?

Reply:
done.